lib/SILOptimizer/Mandatory/TFLowerGraph.cpp

```diff
@@ -671,6 +672,7 @@ static std::string escapeDeclName(DeclName name) {
 /// Produce a "stack trace" for the specified location, producing it in a form
 /// that we can use as a unique op name.
 std::string TFGraphFunctionLowering::getUniqueName(SILDebugLocation loc,
+                                                   const SILFunction *fn,
```

Comment:
in what call-sites would `fn` be different from the member field `SILFn`?

if the former is more general than the latter, should/could we then remove the use of `SILFn`, and make this method static?

Reply:
assuming we still need to pass in `fn` as a param, consider documenting it. it could also be useful to document an example of given some input `baseName`, what the output unique name would look like.

Reply:
Oh, I did not see that it had access to a member field. It should be exactly the same as SILFn. I will make a new PR that changes it to use that.